File: Basics/C#/CSharp.Basics/Samples/GettersAndSetters/Dog.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace CSharp.Basics.Samples.GettersAndSetters
{
    public class Dog
    {
        public string Name { get; set; }
        public string Breed { get; set; }
        public DateTime BirthDate { get; set; }
        public int Age
        {
            get => (DateTime.Now - BirthDate).Days / 365;
        }
    }
}

File: Basics/Kotlin/app/src/main/java/com/suavepirate/kotlinbasics/samples/gettersAndSetters/Dog.kt
package com.suavepirate.kotlinbasics.samples.gettersAndSetters

import java.util.*

class Dog {
    var name: String? = null
    var birthDate: Date = Date()
    var breed: String? = null
    val age: Int
        get() = (((Date().time - birthDate.time) / (1000 * 60 * 60 * 24)) / 365) as Int
}